### PR TITLE
fixed tests

### PR DESCRIPTION
**Pull request by nickwohnhas**
Removed and altered tests so that students can run 'learn' and move on to the next lab.

### CODE REVIEW

**Comment by nickwohnhas** [comment]
@garettarrowood

File: spec/models/author_spec.rb [hunk 1]
```diff
@@ -10,7 +10,3 @@
 #
 
 require 'rails_helper'
-
```
Comment (garettarrowood) [inline spec/models/author_spec.rb]: Since this file contains no tests at all, it can/should be removed.

File: log/test.log [hunk 2]
```diff
@@ -1172,3 +1172,105 @@ Processing by PostsController#edit as HTML
   Rendered posts/edit.html.erb within layouts/application (0.9ms)
 Completed 200 OK in 3ms (Views: 2.9ms | ActiveRecord: 0.0ms)
   [1m[35m (0.9ms)[0m  rollback transaction
+  [1m[36mActiveRecord::SchemaMigration Load (0.1ms)[0m  [1mSELECT "schema_migrations".* FROM "schema_migrations"[0m
```
Comment (garettarrowood) [inline log/test.log]: This file should not have been in the repo to begin with. This repo needs a `.gitignore` file that ignores the `log/` and the `tmp/` directories. It should also ignore `db/test.*` and `db/development.*`.

Before ignoring these, please add a commit that removes all of these files.

**Comment by garettarrowood** [comment]
@nickwohnhas

**Pull request merged**